helm-releases/prometheus.yaml

```diff
@@ -11,17 +11,25 @@ spec:
   chart:
     spec:
       chart: prometheus
-      version: 15.10.1
+      version: 25.1.0
       sourceRef:
         kind: HelmRepository
         name: prometheus
       interval: 10m
   values:
-    pushgateway:
+    prometheus-node-exporter:
```

Comment:
Let's not use the dependent chart then.
It will cause stuttering names like `prometheus-prometheus-node-exporter`. Instead, we should handle `prometheus-node-exporter` in its own helm release. Since it is a standalone chart now.